handlers.py

```diff
@@ -83,6 +87,7 @@ def handle_def(self, bot, update):
         else:
             reply("Please provide a term and it's content to create or update.")
 
+    @spam_protect
```

Comment:
I think we should add it also to the others commands (hello) as user might flood with those messages

Reply:
Sure, I added these to the main ones that do heavy work (db access).